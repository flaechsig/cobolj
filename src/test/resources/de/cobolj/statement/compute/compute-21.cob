IDENTIFICATION DIVISION.
PROGRAM-ID. HELLO-WORLD.


DATA DIVISION.
   WORKING-STORAGE SECTION.
   01 WS-RESULT1 PIC S9(2) VALUE 5.
   
PROCEDURE DIVISION.
    COMPUTE WS-RESULT1 = 50 + 49
    NOT ON SIZE ERROR Display "No Size Error".
    DISPLAY WS-RESULT1

STOP RUN.