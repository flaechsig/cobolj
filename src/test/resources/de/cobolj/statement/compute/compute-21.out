No Size Error
+99
